Assets/MixedRealityToolkit/StandardAssets/Shaders/MixedRealityStandard.shader

```diff
@@ -63,6 +63,8 @@ Shader "Mixed Reality Toolkit/Standard"
         [Toggle(_ROUND_CORNERS)] _RoundCorners("Round Corners", Float) = 0.0
         _RoundCornerRadius("Round Corner Radius", Range(0.0, 0.5)) = 0.25
         _RoundCornerMargin("Round Corner Margin", Range(0.0, 0.5)) = 0.01
+        [Toggle(_INDEPENDENT_CORNERS)] _IndependentCorners("Independent Corners", Float) = 0.0
+        _RoundCornersRadius("Round Corners Radius", Vector) = (0.5,0.5,0.5,0.5)
```

Comment:
small suggestion: it is more readable to have a space after the , here. at first glance, i was seeing a mixture of 0.5 and 5.0 (mistook a comma for a period)

Reply:
suggested change added below

Reply:
```suggestion
        _RoundCornersRadius("Round Corners Radius", Vector) = (0.5, 0.5, 0.5, 0.5)
```